### ISSUE TITLE
ecoport database

### ISSUE THREAD

**Issue by jhammock** [globalbioticinteractions/globalbioticinteractions#513]
I bounced here from the Wayback Machine - it was linked to from an FAO crop database that disappeared a few months ago. Ecoport itself is still standing, but development seems to have stopped in 2006. It describes itself as a relational wiki, and lists 102 contributors (some institutional- the FAO crop database is in there). Database size is hard to guess, but I found more than half a million [references](http://ecoport.org/ep?SearchType=reference&From=1800&To=2010) listed. 

Sample of [aggregated interaction records](http://ecoport.org/ep?SearchType=entityAssociationDisplay&entityId=289&entityRole=HO&coRole=PE)

[Intriguing manifesto](http://ecoport.org/ep?SearchType=displayHelpInfo&type=I&id=101008)

I  can't find any mention of data services. I'm not sure about contacts, but there are some names in their [news section](http://ecoport.org/ep?SearchType=newsDisplay&category=all).

**Comment by jhpoelen** [comment]
@jhammock EcoPort looks pretty neat and much appreciate the manifesto. Did you get a chance to send a message to one of their contact emails like t.putter at ecoport.org or supervisor at ecoport.org?

**Comment by jhammock** [comment]
No, I haven't reached out, myself. Feel free to include me if you do- I suspect GloBI will be in a position to take action before EOL if they're interested in sharing.

**Comment by jhpoelen** [comment]
Ok, just sent an email to supervisor and t.putter .

**Comment by jhpoelen** [comment]
@jhammock have heard back from t.putter. Want to give a try? Perhaps your fancy .si.edu email address helps ; )

**Comment by jhpoelen** [comment]
@jhammock please let me know if you have an idea to continue to pursue indexing this EcoPort database.
